### PR TITLE
HOCS-3477 - FOI Approvals

### PR DESCRIPTION
**Pull request by agraham19**
* Remove defunct workflow stages and screens
* Create generic workflow

### CODE REVIEW

File: src/main/resources/processes/FOI_DRAFT.bpmn [hunk 1]
```diff
@@ -154,118 +151,6 @@
     <bpmn:sequenceFlow id="Flow_08u1iug" sourceRef="Gateway_19rmbk9" targetRef="RESPONSE_TYPE">
       <bpmn:conditionExpression xsi:type="bpmn:tFormalExpression">${ResponseType != "EXEMPTION"}</bpmn:conditionExpression>
     </bpmn:sequenceFlow>
-    <bpmn:sequenceFlow id="Flow_1i0njkg" sourceRef="Gateway_0byag49" targetRef="Gateway_1ma44jq">
-      <bpmn:conditionExpression xsi:type="bpmn:tFormalExpression">${DIRECTION == "FORWARD"}</bpmn:conditionExpression>
-    </bpmn:sequenceFlow>
-    <bpmn:userTask id="SENSITIVITY" name="Sensitivity" camunda:formKey="FOI_SENSITIVITY">
```
Comment (gary-newbolt) [inline src/main/resources/processes/FOI_DRAFT.bpmn:160]: As with in `hocs-data` can you check with Ed that this definitely isn't required please?

**Review by gary-newbolt — COMMENTED** [review]
lgmt... just that one point to check please.

File: src/main/resources/processes/FOI_DRAFT.bpmn (hunk 1 above)
Comment (agraham19) [reply src/main/resources/processes/FOI_DRAFT.bpmn:160]: There's a thread about this in decs-foi on Slack, started at Jul 29th at 10:06 AM. Ed, asked me to remove it and updated the Mural board to reflect this.

Comment (gary-newbolt) [reply src/main/resources/processes/FOI_DRAFT.bpmn:160]: Removal agreed

**Review by gary-newbolt — APPROVED** [review]
lgtm

**Pull request merged**